matlab/gcj2bd.m

```diff
@@ -0,0 +1,21 @@
+function  [bdLat,bdLng]= gcj2bd(gcjLat, gcjLng)
+
+    bdLat = gcjLat;
+    bdLng = gcjLng;
+
+    x_pi = pi * 3000.0 / 180.0;
+    inChina = ~outOfChina(gcjLat, gcjLng);
+    if ~any(inChina),return;end
```

Comment:
this is... violent. consider doing some naive copies and then adding in the deviations from baidu with some `foo(inChina) += blah`. wrap a check-less version maybe? do a `bd_delta`?

Reply:
Not sure what is violent about this ... check if any are actually in China if there are then I apply math only to those?  What am I missing?

Reply:
yeah, don't block all of them out just because of one bad fruit in there. do something like `foo(inChina) = theActualRoutine(something).`

Reply:
This blocks only if all are bad...
Not any in China is logically equivalent to all are out of China.  So the only case that the of statement is true in is where all cases are out of China .

Not a hammer but a surgeons knife only cuts out case where all are outside China

Reply:
oof. my bad again.